Jenkinsfile

```diff
@@ -0,0 +1,98 @@
+#!/bin/env groovy
+
+@Library('cliqz-shared-library@vagrant') _
+
+node('mac-mini-ios') {
+    writeFile file: 'Vagrantfile', text: '''
```

Comment:
lets put that on shared library with some configuration options

Reply:
as the contents of vagrant file is not parameterized, it can be put on the variable in the top of the file, it should improve readability